drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -75,7 +81,8 @@ function DragNDropTemplates(url_name) {
             'draggable': !item.drag_disabled,
             'aria-grabbed': item.grabbed,
             'data-value': item.value,
-            'data-drag-disabled': item.drag_disabled
+            'data-drag-disabled': item.drag_disabled,
+            'aria-describedby': 'Press "Enter", "Space", "Ctrl-m", or "⌘-m" on an item to select it for dropping, then navigate to the zone you want to drop it on.',
```

Comment:
@dovinmu user-facing strings should be internationalized using `gettext` ([example](i.e. https://github.com/edx-solutions/xblock-drag-and-drop-v2/blob/2a9bc5a97c330da54097af348eadbcbc09cf7b18/drag_and_drop_v2/public/js/drag_and_drop.js#L246))